app/src/test/java/org/fossasia/openevent/general/AppLinkUtilsTest.kt

```diff
@@ -0,0 +1,49 @@
+package org.fossasia.openevent.general
+
+import org.fossasia.openevent.general.utils.EVENT_IDENTIFIER
+import org.fossasia.openevent.general.utils.RESET_PASSWORD_TOKEN
+import org.fossasia.openevent.general.utils.AppLinkUtils
+import org.fossasia.openevent.general.utils.VERIFICATION_TOKEN
+import org.fossasia.openevent.general.utils.AppLinkData
+import org.junit.Test
+import org.junit.Assert.assertEquals
+
+private const val EVENT = "event"
+private const val RESET_PASSWORD = "resetPassword"
+private const val VERIFY_EMAIL = "verifyEmail"
+
+class AppLinkUtilsTest {
+
+    private fun getAppLink(type: String): String {
```

Comment:
Can we remove the function and directly write URLs, as I don't think there is any repetition